src/PHP8FunFatal.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/**
+ * Fatal error
+ */
+class C1 {
+	public function method(array $a) {}
+}
+class C2 extends C1 {
+	public function method(int $a) {}
```

Comment:
:no_entry_sign: **Error**: PHP syntax error: Declaration of C2::method(int $a) must be compatible with C1::method(array $a) (*Generic.PHP.Syntax.PHPSyntax*).